Sources/Articles/Documentation.docc/2024/getting-started-with-swift-on-the-server/getting-started-with-swift-on-the-server.md

```diff
@@ -0,0 +1,92 @@
+
+This translates into code that's easy to contribute to, while also being secure and performant. More importantly, it has resulted in libraries that integrate seamlessly with one another.
+
+The ecosystem is governed by the [Swift Server WorkGroup](https://swift.org/server/) (SSWG), a group of developers and companies that are dedicated to upholding and improving Swift's excellent server-side capabilities.
+
+While Apple has a stake in the workgroup, uses it for Private Compute cloud, and is actively contributes a lot, it is by no means the only contributor.
+
+## Where to Start
+
+If you're new to Swift on the Server, you might be wondering where to start.
+
+The first thing you'll need to do is install Swift on your machine. You can do this by downloading the latest version of Swift from the [Swift website](https://swift.org/download/).
+
+From there, you can use various editors to write your Swift code. [Xcode](https://developer.apple.com/xcode/) is the most popular choice on macOS, but you can also use [Visual Studio Code](https://www.swift.org/documentation/articles/getting-started-with-vscode-swift.html), or any other editor that supports a language server.
+
+## Frameworks
+
+Once you have Swift installed, you can start exploring the various frameworks available for building server-side applications. The two major ones are [Hummingbird](https://hummingbird.codes) and [Vapor](https://vapor.codes).
+
+Both are mature, feature-rich frameworks that provide a wide range of tools for building web applications. They also have active communities that are always willing to help out with any questions you might have. Finally, most libraries are compatible with both frameworks, so you can easily switch between them if you need to.
+
+Vapor 4 is an older framework, and has a "batteries-included" approach. It comes with a lot of features out of the box, such as authentication, database support, and templating. This makes Vapor very simple to set up.
+
+Hummingbird 2 is the newer framework, adopting the latest ecosystem tools, and features a more modular approach. This makes Hummingbird more flexible and lightweight. In contrast to Vapor, the modular approach means that you only include the features you need, which can result in a smaller binary size, but requires more setup.
+
+## Creating a Project
+
+Once you've chosen your framework, you can begin a new project. Both Vapor and Hummingbird have templates to get you started:
+
+1. <doc:getting-started-with-hummingbird>
+2. [Getting Started with Vapor 4](https://vapor.codes)
+
+Once you've created your project setup, you can start building your application. Both frameworks have excellent documentation that will guide you through the process of building a web application.
+
+Alongside a web framework, you'll quickly find that you need to use other libraries to build your application. This commonly includes a database, observability and authentication among others.
```

Comment:
Article Needed: What database to pick
Article Needed: Observability
Article Needed: Authentication (JWTKit tutorial for one)